### PR TITLE
[CI] Add Clean meson build for gitaction ci

### PR DESCRIPTION
**Pull request by DonghakPark**
Add Clean meson build .yml file for ci
- This file was taken from nnstreamer's git action and modified to fit nntrainer.
- ref : https://github.com/nnstreamer/nntrainer/blob/main/docs/getting-started.md

**Changes proposed in this PR:**
- .github/workflows/ubuntu_clean_meson_build.yml

Resolves:
- Add gitaction ci

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by jihochu — APPROVED** [review]
LGTM

File: .github/workflows/ubuntu_clean_meson_build.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - name: install minimal requirements
+      run: sudo apt-get update && sudo apt-get install -y gcc g++ pkg-config libopenblas-dev libiniparser-dev libjsoncpp-dev libcurl3-dev tensorflow2-lite-dev nnstreamer-dev libglib2.0-dev libgstreamer1.0-dev libgtest-dev ml-api-common-dev flatbuffers-compiler ml-inference-api-dev libunwind-dev
+    - name: install additional packages for features
+      run: sudo apt-get install -y python3-dev python3-numpy python3
+    - name: install build systems
+      run: sudo apt install meson ninja-build
+    - run: meson setup build/
+      env:
+        CC: gcc
+    - run: meson build
```
Comment (djeong20) [inline .github/workflows/ubuntu_clean_meson_build.yml:31]: quick question. don't we need to add build options used on TAOS/CI?

```suggestion
    - run: meson --buildtype=plain -Denable-tflite-backbone=true (and more) build
```

Comment (DonghakPark) [reply .github/workflows/ubuntu_clean_meson_build.yml:31]: Yes! that's right.
i will fix command as we used in meson build (TAOS)

**Review by djeong20 — APPROVED** [review]
nice work!

**Pull request merged**